diff --git a/bff/go.mod b/bff/go.mod
index 7fc5cb66..73ae0760 100644
--- a/bff/go.mod
+++ b/bff/go.mod
@@ -7,7 +7,7 @@ require (
 	github.com/google/uuid v1.6.0
 	github.com/julienschmidt/httprouter v1.3.0
 	github.com/kubeflow/hub/pkg/openapi v0.0.0
-	github.com/onsi/ginkgo/v2 v2.32.1
+	github.com/onsi/ginkgo/v2 v2.33.0
 	github.com/onsi/gomega v1.43.0
 	github.com/rs/cors v1.11.1
 	github.com/stretchr/testify v1.12.1
